pkg/abstractions/endpoint/autoscaler.go

```diff
@@ -72,7 +72,7 @@ func endpointServeScaleFunc(i *endpointInstance, sample *endpointAutoscalerSampl
 		}
 	}
 
-	if sample.TotalRequests == 0 && exists == 0 {
+	if exists == 0 {
```

Comment:
Are we basically just saying if there is no serve lock, scale the containers to zero?

What happens with the requests that may be in flight?

Reply:
They will just expire which I think is reasonable